app/src/main/java/uk/gov/register/datatype/StringDatatype.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.register.datatype;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.apache.commons.lang3.StringUtils;
+
+public class StringDatatype implements Datatype {
+
+    @Override
+    public boolean isValid(JsonNode value) {
+        return value.isTextual();
+    }
+
+    @Override
+    public boolean hasValue(JsonNode value) {
+        return StringUtils.isNotBlank(value.textValue());
```

Comment:
hmm, there's a difference between an empty string and a string that isn't present.